### ISSUE TITLE
Training settings when training source only models

### ISSUE THREAD

**Issue by zhyx12** [wasidennis/AdaptSegNet#25]
I have trained DeepLab ResNet101 on GTA5，tested on Cityscapes, and got mIOU of 0.3 which is lower than 36.3. I use SGD optimizer with lr 1.0e-5, and poly lr scheduler with max_iter=100K. Can some share the correct training settings?

**Comment by wasidennis** [comment]
@zhyx12 you can use the training setting of our model directly and set all the weights of the adversarial loss to 0. It is basically the one without adaptation. The performance might fluctuate at different iterations, and as we observed, it should reach ~36% within 100k iterations.

**Comment by zhyx12** [comment]
@wasidennis Got it, thanks for your reply! 👍

**Comment by HqWei** [comment]
@zhyx12  The source-only result you get is 30? How many steps have you traind? Have you tried a smaller image size? I am having trouble reproducing the source-only result.

**Comment by lijing1996** [comment]
@wasidennis Setting all the weights of the adversarial loss to 0 is absolutely not without adaptation. The  running stats of batch normalization are updated by the target image during the forward phase even if you set the weight to 0.  I tried to train the model setting the adversarial loss to 0 and got the mIoU about 41%.
